build.sh

```diff
@@ -16,7 +16,16 @@ TAG="nightly"
 LATEST_ONLY=false
 USE_DIGESTS=false
 OFFLINE=false
-DOCKERFILE="./build/dockerfiles/Dockerfile"
+architecture=""
+case $(uname -m) in
+       x86_64) architecture="amd64" ;;
+       s390x) architecture="s390x";;
+esac
+if [ $architecture == "amd64" ]; then \
```

Comment:
hello, I don't see why architecture should be related to the dockerfile that should be used.

Architecture is mostly a parameter/`ARG` of a Dockerfile

Reply:
Well, Dockerfile == alpine, and rhel.Dockerfile == RHEL8. So to build on IBM platforms we need RH OS.

Reply:
or else you need to have TWO build configurations (two dockerfiles) just to support https://wiki.alpinelinux.org/wiki/S390x/Docker and RHEL8/s390x.

Reply:
@benoitf Because the base images which are being used in Dockerfile does not support s390x whereas the ones used in rhel.Dockerfile does. Hence added architecture related code lines.

Reply:
it seems to be hackish

because I could use alpine as well with `https://hub.docker.com/r/s390x/alpine/` so architecture does not imply the system.

Reply:
@benoitf alpine is available for s390x but ```registry.centos.org/centos/httpd-24-centos7``` is not available for s390x.
We tried building che-plugin-registry using the Dockerfile but it failed for the above mentioned image.

Reply:
instead of centos, why not use RHEL?

https://access.redhat.com/containers/?tab=tags&architecture=S390X#/registry.access.redhat.com/rhscl/httpd-24-rhel7

or even better...

https://access.redhat.com/containers/?tab=tags&architecture=S390X#/registry.access.redhat.com/rhel8/httpd-24

(since we use that one in CRW 2.1 downstream)